test/unit/specs/components/Personalization/createAutoRenderingHandler.spec.js

```diff
@@ -12,10 +12,14 @@ governing permissions and limitations under the License.
 
 import {
   CART_VIEW_DECISIONS,
-  PAGE_WIDE_SCOPE_DECISIONS_WITH_DOM_ACTION_SCHEMA_ITEMS,
+  PAGE_WIDE_DECISIONS_WITH_DOM_ACTION_SCHEMA_ITEMS,
   SCOPES_FOO1_FOO2_DECISIONS
 } from "./responsesMock/eventResponses";
 import createAutoRenderingHandler from "../../../../../src/components/Personalization/createAutoRenderingHandler";
+import { isPageWideSurface } from "../../../../../src/components/Personalization/utils/surfaceUtils";
```

Comment:
As mentioned ensure that everything is imported form `./utils`

Reply:
See previous comment on this